alf/examples/mbrl_pendulum.gin

```diff
@@ -48,7 +48,7 @@ Agent.optimizer=@Agent/Adam()
 
 # training config
 TrainerConfig.trainer=@sync_off_policy_trainer
-TrainerConfig.initial_collect_steps=100
+TrainerConfig.initial_collect_steps=1000
```

Comment:
The reason for increase this number is to collect more diverse samples before kick off the training, as only one environment is used.